test/e2e/diagram.navigation.fit.test.ts

```diff
@@ -28,36 +28,36 @@ import { AvailableTestPages, PageTester } from '@test/shared/visu/bpmn-page-util
 import { getBpmnDiagramNames } from '@test/shared/visu/test-utils';
 
 class FitImageSnapshotConfigurator extends ImageSnapshotConfigurator {
-  override getConfig(param: {
+  override getConfig(parameter: {
     fileName: string;
     buildCustomDiffDir: (config: MatchImageSnapshotOptions, fitType: FitType, margin?: number) => string;
     fitType: FitType;
     margin?: number;
   }): MatchImageSnapshotOptions {
-    const config = super.getConfig(param);
-    config.customSnapshotsDir = FitImageSnapshotConfigurator.buildSnapshotFitDir(config.customSnapshotsDir, param.fitType, true, param.margin ? param.margin : 0);
-    config.customDiffDir = param.buildCustomDiffDir(config, param.fitType, param.margin);
+    const config = super.getConfig(parameter);
+    config.customSnapshotsDir = FitImageSnapshotConfigurator.buildSnapshotFitDirectory(config.customSnapshotsDir, parameter.fitType, true, parameter.margin ? parameter.margin : 0);
+    config.customDiffDir = parameter.buildCustomDiffDir(config, parameter.fitType, parameter.margin);
     return config;
   }
 
-  private static buildSnapshotFitDir(parentDir: string, fitType: FitType, withMargin = false, margin?: number): string {
-    const typeDir = join(parentDir, `type-${fitType}`);
+  private static buildSnapshotFitDirectory(parentDirectory: string, fitType: FitType, withMargin = false, margin?: number): string {
+    const typeDirectory = join(parentDirectory, `type-${fitType}`);
```

Comment:
**suggestion:** can we rename the constant into `fitDirectory` to go even further. This will match the method name and make things clearer